subworkflows/local/quality_control/tests/main.nf.test

```diff
@@ -0,0 +1,35 @@
+nextflow_workflow {
+
+    name "Test Subworkflow QUALITY_CONTROL"
+    script "../main.nf"
+    workflow "QUALITY_CONTROL"
+
+    test("test profile") {
+        when {
+            workflow {
+                """
+                input[0] = Channel.of(
+                    [
+                        file(params.pipelines_testdata_base_path + 'nanostring/RCC_files/salmon_01_01.RCC', checkIfExists: true),
+                        file(params.pipelines_testdata_base_path + 'nanostring/RCC_files/salmon_02_02.RCC', checkIfExists: true),
+                        file(params.pipelines_testdata_base_path + 'nanostring/RCC_files/salmon_03_03.RCC', checkIfExists: true),
+                        file(params.pipelines_testdata_base_path + 'nanostring/RCC_files/salmon_04_04.RCC', checkIfExists: true)
+                    ]
+                )
+                input[1] = file(params.pipelines_testdata_base_path + 'nanostring/samplesheets/samplesheet_test.csv', checkIfExists: true)
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert workflow.success},
+                { assert snapshot(
+                    workflow.out.nacho_qc_multiqc_metrics[0].collect { file(it).name }, //undeterministic .png and .txt mqc files
```

Comment:
Why the txt is not stable? Is it possible to sort the results and see if that way they are stable? or it is not because of the content?

Reply:
I can check, I imagine the two .txt files outputted by the subworkflow should be stable. I just thought it wouldn't be worth adding an assertion specifically for those two .txt files, since we are already verifying that all of the files are being generated. I can add it